### PR TITLE
Implement API endpoint fallback for helioviewer

### PR DESCRIPTION
**Pull request by Cadair**
### Description

This should hopefully fix the test suite by using the helioviewer deployment in France.

### CODE REVIEW

**Comment by Cadair** [comment]
@wafels Does this seem sensible?

**Comment by Cadair** [comment]
I am going to tag this as no backport until we get some input from the helioviewer team.

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Looks like the helioviewer.rst file has a doctest that has changed.

**Comment by nabobalis** [comment]
The doctest has revealed that the french source lacks the same full set of source:
```python
odict_items([(('SOHO', 'EIT', None, '171'), 0), (('SOHO', 'EIT', None, '195'), 1), (('SOHO', 'EIT', None, '284'), 2), (('SOHO', 'EIT', None, '304'), 3), (('SOHO', 'LASCO', 'C2', 'white-light'), 4), (('SOHO', 'LASCO', 'C3', 'white-light'), 5), (('SOHO', 'MDI', None, 'magnetogram'), 6), (('SOHO', 'MDI', None, 'continuum'), 7), (('SDO', 'AIA', None, '94'), 8), (('SDO', 'AIA', None, '131'), 9), (('SDO', 'AIA', None, '171'), 10), (('SDO', 'AIA', None, '193'), 11), (('SDO', 'AIA', None, '211'), 12), (('SDO', 'AIA', None, '304'), 13), (('SDO', 'AIA', None, '335'), 14), (('SDO', 'AIA', None, '1600'), 15), (('SDO', 'AIA', None, '1700'), 16), (('SDO', 'AIA', None, '4500'), 17), (('SDO', 'HMI', None, 'continuum'), 18), (('SDO', 'HMI', None, 'magnetogram'), 19), (('STEREO_A', 'SECCHI', 'EUVI', '171'), 20), (('STEREO_A', 'SECCHI', 'EUVI', '195'), 21), (('STEREO_A', 'SECCHI', 'EUVI', '284'), 22), (('STEREO_A', 'SECCHI', 'EUVI', '304'), 23), (('STEREO_B', 'SECCHI', 'EUVI', '171'), 24), (('STEREO_B', 'SECCHI', 'EUVI', '195'), 25), (('STEREO_B', 'SECCHI', 'EUVI', '284'), 26), (('STEREO_B', 'SECCHI', 'EUVI', '304'), 27), (('STEREO_A', 'SECCHI', 'COR1', 'white-light'), 28), (('STEREO_A', 'SECCHI', 'COR2', 'white-light'), 29), (('STEREO_B', 'SECCHI', 'COR1', 'white-light'), 30), (('STEREO_B', 'SECCHI', 'COR2', 'white-light'), 31), (('PROBA2', 'SWAP', None, '174'), 32), (('Yohkoh', 'SXT', None, 'AlMgMn'), 33), (('Yohkoh', 'SXT', None, 'thin-Al'), 34), (('Yohkoh', 'SXT', None, 'white-light'), 35)])
```

What is missing here is all the Hindoe source Ids.

File: docs/guide/acquiring_data/helioviewer.rst [hunk 1]
```diff
@@ -55,63 +55,7 @@ Let us begin by retrieving the available list of sources that Helioviewer suppor
...
-    ('Hinode', 'XRT', 'Any', 'Gband'): 10005
-    ('Hinode', 'XRT', 'Any', 'Open'): 10006
-    ('Hinode', 'XRT', 'Any', 'Ti_poly'): 10007
-    ('Hinode', 'XRT', 'Al_med', 'Any'): 10008
-    ('Hinode', 'XRT', 'Al_poly', 'Any'): 10009
-    ('Hinode', 'XRT', 'Be_med', 'Any'): 10010
-    ('Hinode', 'XRT', 'Be_thin', 'Any'): 10011
-    ('Hinode', 'XRT', 'C_poly', 'Any'): 10012
-    ('Hinode', 'XRT', 'Open', 'Any'): 10013
+    ...
```
Comment (nabobalis) [inline docs/guide/acquiring_data/helioviewer.rst:58]: We did not need to test the entire list anyway.

**Pull request merged**